ops/trigger_http.go

```diff
@@ -67,6 +68,9 @@ func remoteBackupTrigger(args []string) error {
 	if err != nil {
 		return err
 	}
+	if resp.StatusCode != 200 {
+		return fmt.Errorf("Got %d (%s) from the server\n", resp.StatusCode, resp.Status)
```

Comment:
Yeah go's tool chain is very opinionated in some ways. For this particular case we use gofmt tool for formatting - https://golang.org/cmd/gofmt/